hummingbird/ml/convert.py

```diff
@@ -95,7 +95,7 @@ def _convert_sklearn(model, backend, test_input, device, extra_config={}):
     topology = parse_sklearn_api_model(model, extra_config)
 
     # Convert the Topology object into a PyTorch model.
-    hb_model = topology_converter(topology, backend, device, extra_config=extra_config)
+    hb_model = topology_converter(topology, backend, test_input, device, extra_config=extra_config)
```

Comment:
We passed `test_input` into `extra_config` because it is not always used and because some op will require it anyway among the `extra_config`